### PR TITLE
refactor(core): convert instances and loadBalancerFilter.model to TS components

### PR DESCRIPTION
**Pull request by jrsquared**

### CODE REVIEW

File: app/scripts/modules/core/delivery/executionGroup/ExecutionGroup.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class ExecutionGroup extends React.Component<IProps, IState> {
       poll: null,
       canTriggerPipelineManually: !!pipelineConfig,
       canConfigure: !!(pipelineConfig || this.strategyConfig),
-      showAccounts: executionFilterModel.sortFilter.groupBy === 'name',
+      showAccounts: executionFilterModel.asFilterModel.sortFilter.groupBy === 'name',
```
Comment (christopherthielen) [inline app/scripts/modules/core/delivery/executionGroup/ExecutionGroup.tsx]: this makes sense to me now

File: app/scripts/modules/core/domain/loadBalancer.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import { ServerGroup } from './serverGroup';
+import { ILoadBalancer, ILoadBalancerGroup } from './loadBalancer';
```
Comment (jrsquared) [inline app/scripts/modules/core/domain/loadBalancer.ts]: whoops

File: app/scripts/modules/core/instance/instances.component.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+export class InstancesController implements IComponentController {
+  public instances: Instance[];
+  public highlight: string;
+
+  private activeInstance: IActiveInstance;
+  private tooltipTargets: Element[] = [];
+
+  static get $inject(): string[] {
```
Comment (christopherthielen) [inline app/scripts/modules/core/instance/instances.component.ts]: 'ngInject' now

File: app/scripts/modules/core/instance/instances.component.ts [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    return ['$rootScope', '$scope', '$element', '$timeout', '$state'];
+  }
+
+  public constructor(private $rootScope: IRootScopeService,
+                     private $scope: IScope,
+                     private $element: JQuery,
+                     private $timeout: ITimeoutService,
+                     private $state: StateService) {}
```
Comment (christopherthielen) [inline app/scripts/modules/core/instance/instances.component.ts]: goes here:
```
private $state: StateService) {
  'ngInject';
}
```

File: app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.service.ts [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+@autoBindMethods
+export class LoadBalancerFilterService {
+  private isFilterable: (object: any) => boolean;
+  private getCheckValues: (object: any) => string[];
+  private lastApplication: Application;
+
+  constructor(private LoadBalancerFilterModel: LoadBalancerFilterModel,
+              private filterModelService: any,
+              private $log: ILogService) {
+    'nginject';
```
Comment (christopherthielen) [inline app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.service.ts]: I believe this is case sensitive: `ngInject';
Of course I might be wrong.

File: app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.service.ts [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+  }
+
+  private checkSearchTextFilter(loadBalancer: ILoadBalancer): boolean {
+    const filter = this.LoadBalancerFilterModel.asFilterModel.sortFilter.filter;
+    if (!filter) {
+      return true;
+    }
+
+    if (filter.includes('vpc:')) {
+      const [, vpcName] = /vpc:([\w-]*)/.exec(filter);
```
Comment (christopherthielen) [inline app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.service.ts]: neat, I didn't know you could do this.

File: app/scripts/modules/core/instance/instances.component.ts (hunk 3 above)
Comment (jrsquared) [inline app/scripts/modules/core/instance/instances.component.ts]: yep, right out from underneath me

File: app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.service.ts (hunk 6 above)
Comment (jrsquared) [inline app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.service.ts]: Yeah, pretty slick....

File: app/scripts/modules/core/delivery/executionGroup/ExecutionGroup.tsx (hunk 1 above)
Comment (jrsquared) [inline app/scripts/modules/core/delivery/executionGroup/ExecutionGroup.tsx]: this is a reference to this

**Pull request merged**